ResourceStatusSystem/Client/SiteStatus.py

```diff
@@ -43,7 +43,6 @@ def __init__( self ):
     self.__opHelper = Operations()
     self.rssFlag = ResourceStatus().rssFlag
     self.rsClient = ResourceStatusClient()
-    self.wmsAdministrator = RPCClient( 'WorkloadManagement/WMSAdministrator' )
```

Comment:
Really ?? How could that be ? In that case it is a bug in the RPCClient no ? And then how does it help to recreate it every time ?